Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -558,6 +558,19 @@ export class ResourceGroup {
                         return reject(tl.loc("CreateTemplateDeploymentFailed"));
                     }
                     if (result && result["properties"] && result["properties"]["outputs"] && utils.isNonEmpty(this.taskParameters.deploymentOutputs)) {
+                        const setVariablesInObject = (path: string, obj: any) => {
+                            for (var key of Object.keys(obj)) {
+                                if (typeof(obj[key]) === "object") {
+                                    setVariablesInObject(`${path}.${key}`, obj[key]);
+                                }
+                                else {
+                                    tl.setVariable(`${path}.${key}`, JSON.stringify(obj[key]));
```

Comment:
can you please add a console log to print the name of the variable being set ? Similar to L575.